app/plugin_api_test.go

```diff
@@ -510,6 +509,18 @@ func TestPluginAPIRemoveTeamIcon(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestPluginAPIUpdateUserActive(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+	err := api.UpdateUserActive(th.BasicUser.Id, true)
```

Comment:
Could we extend these to actually test if the user was deactivated or not, as well as test what happens when we set an already active user active, etc?

Reply:
@lieut-data Added more tests